language-support/ts/daml-react/tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "es2015"
+     ],
+    "jsx": "react",
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "esModuleInterop": true,
+    "outDir": "lib/",
+    "module": "CommonJS",
+    "declaration": true,
+    "sourceMap": true
+    }
```

We should have a `"files"` or `"include"` entry here.
```suggestion
  }
```